schema/cassandra/temporal/schema.cql

```diff
@@ -94,6 +94,20 @@ CREATE TABLE tasks (
     'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
   };
 
+-- Stores task queue information such as user provided versioning data
+CREATE TABLE task_queue_user_data (
+  namespace_id    uuid,
+  task_queue_name text,
+  data            binary, -- temporal.server.api.persistence.v1.TaskQueueUserData as proto binary blob
+  version         int64,  -- Version of this row, used for optimistic concurrency
+  updated_at      timestamp,
```

Comment:
Why do we need the timestamp here if it's in the blob? For sorting or something?

Including the version column, it's duplicative of the HLC in the blob. Might we only have it here, and not need it in the blob?

Reply:
It can't be used for sorting really..
I initially wanted to use this when seeding the a new cluster with replication tasks to verify the table isn't updated in the meantime but I think it could also be useful as general metadata on the table.

Reply:
Might we remove it from the blob itself then?

Reply:
It's not the same time, here's it's wall clock and on the struct it's a cluster local HLC.
Also note that the HLC only progresses when values are generated due to user interaction, not when applying replication events.

Reply:
I'm also confused at why there are both and what the function of each one is

Reply:
I'll just leave this out and see if we need it later.
I was thinking we could use this timestamp to verify that no new items are added after we seed the replication queue when a new cluster comes online.